src/openforms/tests/e2e/test_input_validation.py

```diff
@@ -966,8 +966,8 @@ def test_unknown_file_type(self):
                 ],
             )
 
-        # Make sure the frontend did not create one:
-        self.assertEqual(TemporaryFileUpload.objects.count(), 1)
+        # Make sure that no temporary files were created
+        self.assertEqual(TemporaryFileUpload.objects.count(), 0)
```

Comment:
I don't really understand what changed here? Why was there initially a temporary file upload and now not anymore?

Reply:
Figured it out - we didn't have a check for empty file name extensions before and we do now, which rejects the temporary file upload created from the frontend.